### ISSUE TITLE
Make the default toolbar buttons inside Viewer class 

### ISSUE THREAD

**Issue by hrghauri** [openseadragon/openseadragon#2210]
This issue is  a result of a discord discussion where I wanted to add custom handler to one of the default toolbar buttons of the viewer. To access one of those buttons I had to use `['..']` notation inside Typescript and @types/openseadragon, I had to write the following code:
```
 (this.viewer1 as any) ['zoomInButton'].addHandler('click', ()=>{
      console.log("zoom in button clicked")
    }); 
```
The above code defeats the purpose of strict type checking.

It would be nice if the default toolbar buttons are publicly available and so can be added @types/openseadragon as well.

**Comment by hrghauri** [comment]
@iangilman Unlike the first place you mentioned where there are lots of public properties and their respective documentation https://github.com/openseadragon/openseadragon/blob/39625aacaa495d79cf1007723c3081bc36113996/src/viewer.js#L94 

I was not able to see any examples of any public properties and their documentation at the following link
`... But for these, I think it's worth doing outside of that (maybe just below it), like here:`
https://github.com/openseadragon/openseadragon/blob/39625aacaa495d79cf1007723c3081bc36113996/src/viewer.js#L191

You recommend doing it differently ? Has something similar been done before?

**Comment by iangilman** [comment]
@hrghauri Yeah, I don't feel comfortable putting the `zoomInButton`, etc. inside the `$.extend` because that would make it something that people could overwrite, which maybe could be an amusing feature, but it would actually take more coding than just adding it in there to make it really work. By putting it just after the `$.extend` you are still applying it to the object, but not in a way that can be overwritten. This would be the first instance of that pattern here in the Viewer, but I think it's worth starting that there. That pattern does exist in other files, like Tile: 

https://github.com/openseadragon/openseadragon/blob/39625aacaa495d79cf1007723c3081bc36113996/src/tile.js#L60

I hope that clears it up... Thank you for asking for clarification!
